spec/support/lib/module_validation.rb

```diff
@@ -102,6 +104,8 @@ def validate_notes_values_are_arrays
 
     def validate_crash_safe_not_present_in_stability_notes
       if rank == Msf::ExcellentRanking && !stability.include?(Msf::CRASH_SAFE)
+        return if stability == Msf::UNKNOWN_STABILITY
```

Comment:
To satisfy the validation of `Msf::CRASH_SAFE` not being present when the module has an excellent ranking, I believe we have two options:

1. Update the validation to skip any modules with an excellent ranking and the new `UNKOWN_STABILITY` sentinel value.
2. Update any existing modules with an excellent ranking and the new `UNKOWN_STABILITY` sentinel value to now have a different ranking.

I have went with option 1, but happy to look into option 2 if we think that is best.